static/src/js/components/AccessMethod/AccessMethod.js

```diff
@@ -663,34 +667,50 @@ export class AccessMethod extends Component {
                       nested
                     >
                       {
-                        selectedVariables.length > 0 && (
+                        !hasVariables && (
                           <p className="access-method__section-status">
-                            {`${selectedVariables.length} ${pluralize('variable', selectedVariables.length)} selected`}
+                            This service has no associated variables.
                           </p>
                         )
                       }
-
                       {
-                        selectedVariables.length === 0 && (
-                          <p className="access-method__section-status">
-                            No variables selected. All variables will be included in download.
-                          </p>
+                        hasVariables && (
```

Comment:
I think you can just consolidate this and the above into a ternary to render if its false or true differently